backend/src/API/memberAPI.ts

```diff
@@ -169,3 +169,89 @@ export const generateMemberArchive = async (
   }
   return MembersDao.generateArchive(membershipChanges, semesters);
 };
+
+/**
+ * Gets the properties of a member given their email.
+ * @param email - the email of a member.
+ * @returns the `MemberProperties` where the email matches the given email.
+ */
+export const getMemberProperties = async (email: string): Promise<MemberProperties | undefined> =>
+  MembersDao.getMemberProperties(email);
+
+/**
```

Comment:
Sorry for the back and forth. This feels very coffee chat specific, so I lean towards putting it in the CoffeeChatAPI.